inc/agent.class.php

```diff
@@ -67,6 +67,16 @@ class PluginFlyvemdmAgent extends CommonDBTM implements PluginFlyvemdmNotifiable
    protected $topic = null;
 
    /**
+    * get mdm types availables
+    */
+   public static function getEnumMdmType() {
+      return array(
+         'android'   => __("Flyve MDM"),
```

Comment:
I began with a type named flyve, and change to android without updating the label. fixing...